router/batchrouter/asyncdestinationmanager/common/common.go

```diff
@@ -13,13 +13,41 @@ import (
 	"github.com/rudderlabs/rudder-server/jobsdb"
 )
 
-type AsyncDestinationManager interface {
+type AsyncUploadAndTransformManager interface {
```

Comment:
```suggestion
var json = jsoniter.ConfigCompatibleWithStandardLibrary

type AsyncUploadAndTransformManager interface {
```

Reply:
We should better use jsoniter and not the stdjson

Reply:
addressed.